deployment/upgradeLive

```diff
@@ -1,18 +1,37 @@
 #!/bin/bash
-# For upgrade just type ./upgradeLive <version>
-# This uploads NVM
+# For upgrade just type ./upgradeLive <version> [replace_script|downgrade|build|install_live_recorder] [ffmpeg build path] [ffmpeg version]
+# in case 'build' is used as parameter then ffmpeg build path and ffmpeg version are mandatory
+
 #set -e
-USAGE="usage: upgradeLive [version] <replace_script|downgrade>"
-[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh" || (echo "nvm not found in $NVM_DIR, this is a must, Exiting!" ; exit 1 )
+
+PRODUCT_PATH=/opt/kaltura/liveController
```

Comment:
This should be either a token or you can calculate the real path based on your current location.